### ISSUE TITLE
A11y_SQL Azure Data Studio_Table_General_ScreenReader: The voiceover is not announcing the listed item numbers for the combo-box 'Schema', 'Type', 'Durability' associated with 'General' tab.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19564]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureDataStudiofordesktopMAC_May2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#Voiceover;#AINotInScope;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: 1.36.1
Mac Version: macOS Monterey 12.4
Screen Reader: Voiceover

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure Data Studio and connect to server.
3. Tab to 'SERVERS' section and expand.
4. Tab till 'Databases' menu item and expand.
5. Select any of the databases and expand.
6. Tab till 'Tables' sub-menu item.
7. Right click on table sub-menu item and click 'New Table'.
8. Tab till 'General' tab.
9. Tab to 'Schema' combo-box.
10. Turn on voiceover and observe.

## Actual:
The voiceover is not announcing the listed item numbers for the combo-box 'Schema', 'Type', 'Durability' associated with 'General' tab.

## Expected:
The voiceover should announce the listed item numbers for the combo-box 'Schema', 'Type', 'Durability' associated with 'General' tab. For each of the listed items the voiceover should announce its listed number.

## User Impact:
Screen reader users will find it difficult in knowing the number of listed items to choose from the combo-box.

## Attachment:
![7_Listed Numbers](https://user-images.githubusercontent.com/98393966/170928228-e14e5cda-8927-44d4-baab-d6a65a86d9c4.gif)

**Comment by alanrenmsft** [comment]
this is an issue with voice over, it is not reading the index and count of items in the native html select element.

**Comment by bbarik06** [comment]
Since the reported issue is an expected behavior. Thus, closing the bug.
